woocommerce/woocommerce-ios
# Hack Week: Today Stats Widget (Step 1)

This PR completes the first step of the Stats widget Project as defined here: p91TBi-2Vw

## Changes
- Added a today stats extension to the app that displays the current day stats, if v4 stats are available for the site.
- An error message is displayed in the widget if the user is not logged in to the app.
- An error message is displayed in the widget if the store does not have WC-Admin installed or if WooCommerce 4.0 is not installed.

## Testing
1. Run the Woo app, and make sure you are not logged.
2. **Open the widget ->** Make sure you see an error about your login state.
3. Login in the Woo app, and choose an app that doesn't support stats v4.
4. **Open the widget ->** Make sure you see an error on stats v4 not available.
5. Login in the Woo app, and choose an app that supports stats v4.
6. **Open the widget ->** You will be able to see the stats on the widget, and the selected website.
7. Open the app, and change the store.
8. **Open the widget ->** Verify that the widget data is refreshed when store is changed.

## Notes for testing
The widget is disabled by default now (because it's not already completed).
To enable it:
1. Click the project file in the project navigator
2. Click the containing app target (WooCommerce)
3. Click **Build Phases** tab
4. Open **Embed App Extensions**
5. Add the extension (TodayStatsWidget.appex)

<img width="881" alt="Schermata 2020-06-19 alle 11 51 12" src="https://user-images.githubusercontent.com/495617/85138221-b99ed000-b242-11ea-9bfb-6162a7d27d9d.png">


## Screenshots
| Dark Mode off            |  Dark mode on | Error state
:-------------------------:|:-------------------------:|:-------------------------:
![Simulator Screen Shot - iPhone 11 Pro - 2020-06-15 at 12 42 13](https://user-images.githubusercontent.com/495617/84648809-d2874880-af05-11ea-97b9-816fef4d1ad0.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-06-15 at 12 42 26](https://user-images.githubusercontent.com/495617/84648859-e59a1880-af05-11ea-83d6-533ebb9909e1.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-06-15 at 12 40 38](https://user-images.githubusercontent.com/495617/84648876-e92d9f80-af05-11ea-9957-ccc3230dc495.png)



Update release notes:
